paasta_tools/api/api.py

```diff
@@ -148,6 +148,10 @@ def make_app(global_config=None):
         "service.instance.tasks.task",
         "/v1/services/{service}/{instance}/tasks/{task_id}",
     )
+    config.add_route(
+        "service.instance.remote_run",
+        "/v1/services/{service}/{instance}/remote_run",
+    )
```

Comment:
for how the authz design works, it'd be easier if this wasn't under the `/v1/services` umbrella